content/community/contributing-to-documentation.md

```diff
@@ -123,20 +94,24 @@ A language select will be displayed if the page specifies the following in the f
  - java
  - javascript
  - ruby
+ - kotlin
  - dotnet
+ - kotlin
  ```
 
 The following languages are currently supported:
 
 * Java
 * JavaScript
 * Ruby
+* Kotlin (optional)
 * .Net (optional; some pages only)
+* Kotlin (optional)
```

Comment:
Is there a reason this is mentioned twice above and here?

Reply:
One is an example of the polyglot front matter, and one is a list of the currently supported languages. Is that what you mean?

Reply:
I think he means, why does kotlin occur four times in two lists? Shouldn't it occur only twice in two lists?

Reply:
Yes, found this too. Fixed it ;)